Issue #2824348

jonathangreen opened the pull request:
Updating the patch from Drupal.org and making pull request. Issue on Drupal.org:
https://www.drupal.org/node/2824348

jonathan1055 commented:
Thanks @jonathangreen for doing this. Now atleast we can see the tests running and investigate the failure.

jonathangreen committed:
Issue #2824348 by dougmorin0, samuel.mortenson, jonathan1055, simonmc, jonathan.green: Warnings when using token replacements in multiple string context parameters

jonathangreen commented:
@fago I updated the patch from d.o [here](https://www.drupal.org/node/2824348#comment-12047585) and updated the prophecy so that it reflects the new call to `getContextDefinition` in the patch. Should be passing all tests now.
